Automattic/wp-calypso
# Jetpack Cloud Cart Integration - Update Cart styles to match the design

#### Proposed Changes
- Add a New Cart Icon component to override the default icon provided by masterbar
- Add a New Count component that renders the count next to the cart icon
- Fix styles inside the cart popover to match the design

#### Testing Instructions
*Note: [Figma Design](i45eiTwhKnMEnDtn86U6IR-fi-2993%3A32699&t=CdumeBBITUkseOx8-0)* 

* Make sure you are logged-in to WordPress.com 
* click on Jetpack Cloud live link below
* or boot up this PR 
    * Run `git fetch && git checkout fix/style-for-cloud-cart`
    * Run `yarn start-jetpack-cloud`
    * Goto http://jetpack.cloud.localhost:3000
* you will be presented with a landing page, asking you to select a site, choose any site from the list
* Once selected, you will be redirected to `/backup/:site-slug` page by default, replace `backup` with `pricing` and you should land in pricing page
* Now append this query string (`?flags=jetpack/pricing-page-cart`) to the end of URL to enable the cart and reload the page
* Make sure the cart shows up in the header
* Switch to Products section and confirm the following
    * Confirm that the cart icon in the header matches the design
    * Cart count should be hidden when there is no item in the cart
    * Now add any products to the cart
    * Click on the cart icon to open the cart popover and confirm that it matches the cart design
    * Also confirm that the cart count is updated to match the total items in the cart
    * Switch to mobile view and make sure it matches the design
    

#### Before
![Screenshot 2022-12-16 at 2 56 45 PM](https://user-images.githubusercontent.com/2027003/208066995-e23f17be-23c6-4a76-bf8b-2befc093c6b8.png)

#### After
![Screenshot 2022-12-16 at 2 55 50 PM](https://user-images.githubusercontent.com/2027003/208067029-c6dc19fa-a5e0-48ac-ab5b-bc9595850943.png)



#### Pre-merge Checklist





Related to #


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1203534015891189
  - 0-as-1203534015891169
